components/widgets/title/index.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components'
```

Comment:
Can you add the widget to the README, please?

Reply:
Done.